dataland-frontend/src/components/pages/SearchTaxonomy.vue

```diff
@@ -2,49 +2,166 @@
   <AuthenticationWrapper>
     <TheHeader />
     <TheContent>
-      <SearchTaxonomyHeader :scrolled="scrolled" />
-      <EuTaxoSearchBar
-        @autocomplete-focus="handleAutoCompleteFocus"
-        :stockIndices="stockIndices"
-        ref="euTaxoSearchBar"
-        @scrolling="handleScrolling"
-      />
+      <div
+        class="col-12 bg-white"
+        :class="[searchBarToggled && pageScrolled ? ['d-search-toggle', 'fixed'] : '']"
+        ref="searchBarAndIndexTabContainer"
+      >
+        <SearchTaxonomyHeader />
+        <MarginWrapper>
+          <EuTaxoSearchBar
+            v-model="currentInput"
+            ref="euTaxoSearchBar"
+            :taxoSearchBarName="taxoSearchBarName"
+            @companies-received="handleCompanyQuery"
+            @rendered="handleEuTaxoSearchBarRender"
+          >
+          </EuTaxoSearchBar>
+
+          <div
+            :class="[
+              pageScrolled && !searchBarToggled
+                ? ['col-12', 'align-items-center', 'grid', 'bg-white', 'd-search-toggle', 'fixed']
+                : '',
+            ]"
+          >
+            <span v-if="!searchBarToggled && pageScrolled" class="mr-3 font-semibold">Search EU Taxonomy data</span>
+            <Button
+              v-if="!searchBarToggled && pageScrolled"
```

Comment:
Das würde ich noch etwas umstrukturieren: Das v-if kommt an das umgebende div, und das umgebende div enthält die dynamischen Klassen statisch.

Reply:
Aber das umgebende div beinhaltet auch die IndexTabs.   Die dürfen kein v-if bekommen, weil die immer da sein müssen. Sie ändern nur ihre class.

Wenn man es hingegen auf zwei divs aufspaltet, kann man das v-if für Button/Icon/Text in das erste div mit reinnehmen, aber dadurch muss man duplizierten Code in die Klassen der zwei divs schreiben.
Ich habe das mal ausprobiert:

![image](https://user-images.githubusercontent.com/99255296/179529732-9ad358b9-d1ae-410e-932b-0c36579eb5d5.png)

Im Endeffekt spart das dann keine LoC.
Was meinst du?

Reply:
würde das gerne morgen mit dir mal ausprobieren - ich glaube da war ich nicht ganz richtig. Ich glaube, dass jetzt das scroll-Verhalten von der Überschrift "Search EU Taxonomy data" anders wird - lass uns das ansehen - und entscheiden was uns besser gefällt.

Wenn es aber deine Lösung wird: 
a) statt `:class` nur `class` wenn es keine dynamik gibt
b) IndexTabs braucht nicht noch in ein div verpackt werden - dadurch gewinnt man ja nix.

Reply:
Alles klar, gerne 👍

Reply:
ok